src/components/SearchForm.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import NamePicker from './NamePicker';
+import LifeEventPicker from './LifeEventPicker';
+import RelationshipPicker from './RelationshipPicker';
+import CategoryPicker from './CategoryPicker';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { makeStyles } from '@material-ui/core/styles';
+
+export default function SearchForm(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="md">
+      <form className={classes.form} noValidate>
+        <Typography component="h1" variant="h5">
+          Search our society's records
+        </Typography>
+        <NamePicker />
+        <LifeEventPicker />
+        <RelationshipPicker />
+        <CategoryPicker />
+        <div style={{ marginTop: 20 }}>
+          <Button variant="outlined" color="primary" onClick={() => console.log('Submit')}>
```

Comment:
I think you need to add `type='submit'` for it to submit the form when clicked. In that case, you won't need the `onClick` event but will just respond to the form's `onSubmit` event.

Reply:
Updated to type='submit', thanks!